tee-worker/ts-tests/integration-tests/vc_ii.test.ts

```diff
@@ -0,0 +1,163 @@
+import { randomBytes, KeyObject } from 'crypto';
```

Comment:
This should be direct invocation, not indirect invocation test right ?
If yes you will need to change the file name.